Testscripts/Windows/Power-Hibernate.ps1

```diff
@@ -168,44 +179,44 @@ done < netdev.log
 		Run-LinuxCmd -ip $AllVMData.PublicIP -port $AllVMData.SSHPort -username $user -password $password -command "./test.sh" -runAsSudo -RunInBackground -ignoreLinuxExitCode:$true | Out-Null
 		Write-LogInfo "Sent hibernate command to the VM and continue checking its status in every 15 seconds until 2 minutes timeout "
 
-		# Verify the VM status
-		# Can not find if VM hibernation completion or not as soon as it disconnects the network. Assume it is in timeout.
-		$timeout = New-Timespan -Minutes 10
-		$sw = [diagnostics.stopwatch]::StartNew()
-		while ($sw.elapsed -lt $timeout){
-			Wait-Time -seconds 15
-			$vmStatus = Get-AzVM -Name $vmName -ResourceGroupName $rgName -Status
+			# Verify the VM status
```

Comment:
indent

Reply:
Fixed in ae66ef26caf39800d996c15cf19eed2b44ecfc9f